_pages/membership.md

```diff
@@ -8,9 +8,9 @@ js-package: membership
 css_bundle: membership
 layout: flow
 jumbotron:
-  image: /assets/images/content/memberhsip_bg.jpg
   title: Shape the future of Arm Software
   description: ''
+  image: /assets/images/content/memberhsip_bg.jpg
```

Comment:
Is that really the correct filename? It is misspelt ...

Reply:
![](http://linaro.org/assets/images/content/memberhsip_bg.jpg)

Reply:
Oh well.